docs/pages/architecture/agents.mdx

```diff
@@ -0,0 +1,265 @@
+  cluster's certificate authority, handles user authentication/authorization and
+  issues short-lived client certificates.
+- **Teleport agents:** Agents perform authentication against infrastructure
+  resources, proxy user traffic to those resources, and perform protocol
+  parsing.
+- **Teleport client tools (`tsh`, Teleport Connect, and the Teleport Web UI):**
+  Connect to resources in your infrastructure through Teleport agents and the
+  Teleport Proxy Service.
+- **Local proxies:** In some cases, `tsh` spins up local proxy servers that
+  authenticate to Teleport and forward traffic from client tools (e.g., database
+  clients and AWS SDK applications).
+- **Infrastructure resources:** Teleport can protect access to self-hosted
+  infrastructure as well as infrastructure managed by a cloud provider.
+
+## Retrieving credentials from the Teleport Auth Service
+
+The Teleport Auth Service issues certificates that enable users to communicate
+with Teleport agents and for Teleport agents to communicate with resources in
+your infrastructure.
+
+### Credentials for Teleport agents
+
+The Teleport Auth Service runs a certificate authority that issues a host
+certificate to an agent when it joins the cluster for the first time. Read [Join
+Services to your Teleport Cluster](../agents/join-services-to-your-cluster.mdx)
+for the available methods you can use to join an agent to your Teleport cluster.
+
+All agents in a Teleport cluster keep the Auth Service updated on their status
+with periodic ping messages. They report their IP addresses and the values of
+their assigned labels. Clients can access the list of all agent instances in
+their cluster via the Auth Service API or CLI.
+
+An agent's identity is represented by SSH host certificate it receives after
+registering withing the cluster:
+
+![Host certificate](../../img/architecture/ssh-host-cert@1.2x.svg)
+
+This certificate contains information about the agent, including:
+
+- The **host ID**, a generated UUID unique to an agent.
+- A **hostname**, which defaults to the return value of the `hostname` command
+  on the agent host, but can be configured.
+- The **cluster name**. On cloud-hosted Teleport Enterprise clusters, this is a
+  subdomain of `teleport.sh`. On self-hosted clusters, the default value is the
+  `hostname` of the Teleport Auth Service, and users can configure this.
+- The agent's **role** (e.g., `node,proxy`) encoded as a certificate extension. 
+- The **expiry time** of the certificate.
+
+### Credentials for Teleport clients
+
+When a user logs into the cluster with the `tsh login` command, the Teleport
+Auth Service uses the `user` certificate authority to sign the certificate. See
+[Issuing User Certificates](../architecture/authentication.mdx) for more details
+on how it works. If the user authenticates via the Teleport Web UI, the browser
+retrieves a session cookie.
+
+In most cases, Teleport users must retrieve another user certificate in order to
```

Comment:
Not sure what "another" means. Another, as in beyond the one retrieved via the `tsh login` command?

Reply:
That's what I meant, e.g., `tsh kube login`. Would that be correct?